lightkurve/tests/test_lightcurve.py

```diff
@@ -442,6 +442,60 @@ def test_flatten_robustness():
 
 
 @pytest.mark.remote_data
+def test_lightcurvecollection_create():
+    
+    lcf_1 = KeplerLightCurveFile.from_archive('Kepler-10b', quarter=1).PDCSAP_FLUX
+    lcf_2 = KeplerLightCurveFile.from_archive('Kepler-8b', quarter=3).PDCSAP_FLUX
+    lcf_3 = KeplerLightCurveFile.from_archive('Kepler-16b', quarter=7).PDCSAP_FLUX
+
+    lcc = LightCurveCollection((lcf_1, lcf_2))
+
+    assert(lcc.data[lcf_1.keplerid][0] == lcf_1)
+    assert(lcc.data[lcf_2.keplerid][0] == lcf_2)
+
+@pytest.mark.remote_data
+def test_lightcurvecollection_append():
+    
+    lcf_1 = KeplerLightCurveFile.from_archive('Kepler-10b', quarter=1).PDCSAP_FLUX
+    lcf_2 = KeplerLightCurveFile.from_archive('Kepler-8b', quarter=3).PDCSAP_FLUX
+    lcf_3 = KeplerLightCurveFile.from_archive('Kepler-16b', quarter=7).PDCSAP_FLUX
+
+    lcc = LightCurveCollection((lcf_1, lcf_2))
+    lcc.append(lcf_3)
+    assert(lcc.data[lcf_3.keplerid][0] == lcf_3)
+
+@pytest.mark.remote_data
+def test_lightcurvecollection_length():
+    lcf_1 = KeplerLightCurveFile.from_archive('Kepler-10b', quarter=1).PDCSAP_FLUX
+    lcf_2 = KeplerLightCurveFile.from_archive('Kepler-8b', quarter=3).PDCSAP_FLUX
+    lcf_3 = KeplerLightCurveFile.from_archive('Kepler-16b', quarter=7).PDCSAP_FLUX
+
```

Comment:
I don't understand why these tests are duplicates of above? Let's keep it in one test function to reduce lines.

Reply:
Done!